resources/assets/js/app/models/UsersVoteModel.js

```diff
@@ -0,0 +1,7 @@
+var baseModel = require('../instances/Model');
+
+module.exports = baseModel.extend({
+    url: function () {
+        return 'users/' + this.model.get('id')+'/votes';
+    }
+});
```

Comment:
you no need such Models, same as in controller intself, handle those difference with parentUrl